src/modules/fw_att_control/FixedwingAttitudeControl.cpp

```diff
@@ -249,22 +285,23 @@ FixedwingAttitudeControl::parameters_update()
 void
 FixedwingAttitudeControl::vehicle_control_mode_poll()
 {
-	bool vcontrol_mode_updated;
+	bool updated = false;
 
 	/* Check if vehicle control mode has changed */
-	orb_check(_vcontrol_mode_sub, &vcontrol_mode_updated);
-
-	if (vcontrol_mode_updated) {
+	orb_check(_vcontrol_mode_sub, &updated);
 
+	if (updated) {
 		orb_copy(ORB_ID(vehicle_control_mode), _vcontrol_mode_sub, &_vcontrol_mode);
 	}
 }
 
 void
 FixedwingAttitudeControl::vehicle_manual_poll()
 {
-	// only update manual if in a manual mode
-	if (_vcontrol_mode.flag_control_manual_enabled) {
+	bool manual_updated = false;
+	orb_check(_manual_sub, &manual_updated);
```

Comment:
@RomanBapst I believe this change was done intentionally for flaps deployment. I'll track down the original so we can make sure the intended fix is preserved.

Reply:
https://github.com/PX4/Firmware/pull/9607